Kickstarter-iOS.playground/Pages/Color Palettes.xcplaygroundpage/Contents.swift

```diff
@@ -106,16 +106,16 @@ let dropShadowView = colorBlockStackView(color: .ksr_dropShadow, colorName: "Dro
 
 
 let gradient1 = gradientBlockStackView(colorName: "Lavender / Powder",
-                                       startColor: .ksr_lavenderToPowderGradientStart,
-                                       endColor: .ksr_lavenderToPowderGradientEnd)
+                                       startColor: .ksr_grey_500,
```

Comment:
🤔 I don't even remember those old color names. Did you pick random colors or are these the same?

Reply:
Since we don't have the `lavender` or `powder`, I just picked random colors, so the playground could work again.